CHANGELOG.md

```diff
@@ -13,6 +13,11 @@ The REST API versions in the table are just for your information as the values a
 |\>= 10.0.0|3
 |<= 9.0.1|2
 
+<a name="12.5.4"></a>
+## 2020-04-20 Version [12.5.4](https://github.com/gooddata/gooddata-js/compare/v12.5.3...v12.5.4)
```

Comment:
where is version `12.5.3`?

Please take a look at the order of versions in the previous link, 
it is from `v12.5.2` => `v12.5.1` (newer to older version)
`https://github.com/gooddata/gooddata-js/compare/v12.5.2...v12.5.1`

Reply:
- previous link is incorrect - please check the second previous link
- 12.5.3 is latest version was released which isn't mentioned in CHANGELOG.md

Reply:
Yes, v12.5.3 is [here](https://www.npmjs.com/package/@gooddata/gooddata-js/v/12.5.3) on npm registry